package-lock.json

```diff
@@ -5143,17 +5143,6 @@
         "@rdfjs/data-model": "^1.1.1"
       }
     },
-    "@sentry/browser": {
```

Comment:
Seems weird to have these changes in package-lock.json, when there are no changes in package.json. Sure this is right?